### PR TITLE
Feat: Home page

### PR DESCRIPTION
**Pull request by sri-rishi**
Home page:

- Navbar 
- Categories
- Popular Quizzes
- Footer

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks good to merge

File: components_styles/home.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    max-height: 12rem;
+    height: 100%;
+}
+
+.quiz-div-img {
+    height: 100%;
+    object-fit: cover;
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+}
```
Comment (Ankur9669) [inline components_styles/home.css:86]: Suggestion: you can used shorthanf border-radius here

**Review by Ankur9669 — APPROVED** [review]
UI suggestion: 
image
You can add cursor pointer to the hamburger icon

File: components_styles/home.css (hunk 1 above)
Comment (sri-rishi) [reply components_styles/home.css:86]: done

**Comment by sri-rishi** [comment]
> UI suggestion: image You can add cursor pointer to the hamburger icon

done

**Pull request closed without merging**